cmd/agent/Dockerfile

```diff
@@ -1,16 +1,27 @@
-FROM golang:1.15 as build
+FROM golang:1.15.3-buster as build
 COPY . /src/agent
 WORKDIR /src/agent
 ARG RELEASE_BUILD=true
 ARG IMAGE_TAG
-RUN apt-get update && apt-get install -qy libsystemd-dev
+
+# Backports repo required to get a libsystemd version 246 or newer which is required to handle journal +ZSTD compression
+RUN echo "deb http://deb.debian.org/debian buster-backports main" >> /etc/apt/sources.list
+RUN apt-get update && apt-get install -t buster-backports -qy libsystemd-dev
+
 RUN make clean && make IMAGE_TAG=${IMAGE_TAG} RELEASE_BUILD=${RELEASE_BUILD} BUILD_IN_CONTAINER=false agent
 
 FROM debian:buster-slim
+
+# Backports repo required to get a libsystemd version 246 or newer which is required to handle journal +ZSTD compression
+RUN echo "deb http://deb.debian.org/debian buster-backports main" >> /etc/apt/sources.list
+RUN apt-get update && apt-get install -t buster-backports -qy libsystemd-dev && \
+  rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
```

Comment:
you could maybe omit this since it gets done again in the next install?

Reply:
hmm, do we need the newer libsystemd version at build time or run time? (or both?)

Reply:
sorry, i was specifically referring to the `rm -rf..... ` part

Reply:
Oh I see, we probably want to remove it both times to keep each individual build layer small

Reply:
Ah good catch, I should fix this in promtail